### PR TITLE
Factory recive transaction

### PR DESCRIPTION
**Pull request by ferchuhill**
change from motive to reason and how to check if the role exist

### CODE REVIEW

File: src/cloud/services/RecyclingCenterService.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ const editRecyclingCenter = async (
 
   if (role) {
     await role.fetch();
-    if (role.get('name') !== name) {
+    if (role.get('name') !== nameNormalize) {
```
Comment (santiagosemhan) [inline src/cloud/services/RecyclingCenterService.ts:59]: normalizedName?

Comment (ferchuhill) [reply src/cloud/services/RecyclingCenterService.ts:59]: how we store the name of the role for that factory, e.g.: Itambe Mini -> itambe_mini

Comment (ferchuhill) [reply src/cloud/services/RecyclingCenterService.ts]: change the variable name

File: src/cloud/services/RecyclingCenterService.ts [hunk 2]
```diff
@@ -51,16 +51,16 @@ const editRecyclingCenter = async (
   if (!recyclingCenter) {
     throw new Error(`RecyclingCenter ${id} not found`);
   }
-  const nameNormalize = normalizeRoleName(`ROL_RC_${name}`);
+  const normalizeName = normalizeRoleName(`ROL_RC_${name}`);
```
Comment (santiagosemhan) [inline src/cloud/services/RecyclingCenterService.ts:54]: normalizedName 👯‍♂️

Comment (ferchuhill) [reply src/cloud/services/RecyclingCenterService.ts]: done

**Review by santiagosemhan — APPROVED** [review]

**Pull request merged**